src/resources/SecurityCache/SecurityCacheInterfaces.ts

```diff
@@ -0,0 +1,100 @@
+export interface SecurityCacheScheduleModel {
+    enabled?: boolean;
+    frequency?: {
+        hourMinute?: {
+            hour?: number;
+            minute?: number;
+        };
+        monthDay?: number;
+        seed?: number;
+        type?: string;
+        value?: number;
+        weekDays?: [string];
+    };
+    id?: string;
+    organizationId?: string;
+    refreshType?: string;
+    resourceId?: string;
+    scheduleType?: string;
+}
+
+interface Status {
+    numberOfEntitiesInError?: number;
+    numberOfEntitiesProcessed?: number;
+    refreshType?: string;
+    totalNumberOfEntities?: number;
+    type?: string;
+}
+
+interface Statistics {
```

Comment:
should this be exported?

Reply:
I did not export It because It is not used outside this file.